content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -353,26 +353,35 @@ Enable the database monitoring propagation feature by setting the following envi
 
 {{< /tabs >}}
 
-## Explore the APM Connection
+## Explore the APM Connection in DBM
 
 ### Attribute active database connections to the calling APM services
 
 {{< img src="database_monitoring/dbm_apm_active_connections_breakdown.png" alt="View active connections to a database broken down by the APM Service they originate from.">}}
 
 Break down active connections for a given host by the upstream APM services making the requests. You can attribute load on a database to individual services to understand which services are most active on the database. Pivot to the most active upstream service’s service page to continue the investigation.
 
+### Filter the Databases List for specific APM calling services.
+
+{{< img src="database_monitoring/dbm_filter_by_calling_service.png" alt="Filter the Databases List for specific APM calling services.">}}
+
 ### View the associated trace for a query sample
 
 {{< img src="database_monitoring/dbm_query_sample_trace_preview.png" alt="Preview the sampled APM trace that the query sample being inspected was generated from.">}}
 
 When viewing a Query Sample in Database Monitoring, if the associated trace has been sampled by APM, you can view the DBM Sample in the context of the APM Trace. This allows you to combine DBM telemetry, including the explain plan and historical performance of the query, alongside the lineage of the span within your infrastructure to understand if a change on the database is responsible for poor application performance.
 
-### Identify the downstream database hosts of APM services
+## Explore the DBM Connection in APM
+
+### Visualize the downstream database hosts of APM services
 
 {{< img src="database_monitoring/dbm_apm_service_page_db_host_list.png" alt="Visualize the downstream database hosts that your APM Services depend on from the Service Page.">}}
 
 On the APM Service Page, view the direct downstream database dependencies of the service as identified by Database Monitoring. Quickly determine if any hosts have disproportionate load that may be caused by noisy neighbors.
 
+### Identify inefficiencies using explain plans for database queries in traces 
```

Comment:
I feel like there's a lot more going on in this image/tab than the explain plans. Maybe also needs an explainer section that talks about being able to view query info and database health info within trace panel?